src/conv/k-g/k_parser.cpp

```diff
@@ -39,7 +39,8 @@ enum class KState {
     Node,
     Element_Shell,
     Part,
-    Section_Shell
+    Section_Shell,
+    PART_ADAPTIVE_FAILURE

```

Comment:
Please, follow the style of the other states, i.e. write *Part_Adaptive_Failure* and put it below *Part*. It doesn't matter, if the states below getting other numbers.